packages/editor/templates/Crypto Pizzas/contracts/CryptoPizza.sol

```diff
@@ -1,89 +1,107 @@
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.10
 pragma solidity ^0.5.10;
 
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
-import "../node_modules/openzeppelin-solidity/contracts/token/ERC721/IERC721Receiver.sol";
-import "../node_modules/openzeppelin-solidity/contracts/introspection/ERC165.sol";
-import "../node_modules/openzeppelin-solidity/contracts/math/SafeMath.sol";
+// Imports symbols from other files into the current contract.
+// In this case, a series of helper contracts from OpenZeppelin.
+import "../node_modules/@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "../node_modules/@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "../node_modules/@openzeppelin/contracts/introspection/ERC165.sol";
+import "../node_modules/@openzeppelin/contracts/math/SafeMath.sol";
 
+// A contract is a collection of functions and data (its state)
+// that resides at a specific address on the Ethereum blockchain.
+// "is" keyword inherits functions and  keywords from two external contracts
```

Comment:
Perhaps include reference to inheritance? https://solidity.readthedocs.io/en/v0.6.2/contracts.html#inheritance

Reply:
It's kind of the same as the link above, but I reworded to be clearer